apps/condo/package.json

```diff
@@ -79,6 +79,7 @@
     "jwt-decode": "3.1.2",
     "less": "^3.12.2",
     "lodash": "^4.17.21",
+    "micro-memoize": "^4.0.9",
```

Comment:
Do we really need another one memorize lib?

Reply:
No, I will delete it